modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -282,7 +296,7 @@ You can send a test message to verify connectivity between Runtime Fabric and th
 .. Trigger a test message for the selected service.
 .. Verify the text that is forwarded to log forwarding solution.
 
-== Disable Anypoint Monitoring or Output Plugin Log Forwarding
+== Disable Anypoint Monitoring or External Log Forwarding per Application
```

Comment:
I think we're missing a step here where we need the user to deploy or re-deploy their application.
@hegderohit89 to confirm.

Reply:
Ya thats right, i will add that